src/js/components/todoItem.js

```diff
@@ -0,0 +1,22 @@
+import { VIEW, EDIT, COMPLETE } from '../constant/constant.js';
+
+export class TodoItem {
+  constructor(state, title, index) {
+    this.state = state;
+    this.title = title;
+    this.index = index;
+  }
+  template() {
+    const classTemplate = this.state === COMPLETE ? 'class="completed"' : '';
+    const checkedTemplate = this.state === COMPLETE ? 'checked' : '';
+    let template = `<li ${classTemplate} data-index=${this.index}>
```

Comment:
```suggestion
    const template = `<li ${classTemplate} data-index=${this.index}>
```
let보다는 const가 더 좋아보입니다.